index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="heading">
+        <a class="logo" href="#"> 
+            John Doe
+        </a>
+        <a href="#">
+            <img class="menu" src="./assets/menu_white_24dp.svg" alt="menu_bar">
+        </a>
```

Comment:
- [ ] For the hamburg icon, enclose it within `nav` tags. Remember it still has to be clickable

Reply:
I have made the changes. Thank you so much.